src/ticketmap/index.tsx

```diff
@@ -448,13 +501,16 @@ export default class TicketMap extends Component<Props, State> {
   getDefaultColor(ticketGroups: TicketGroup[]) {
     const { sectionPercentiles } = this.props
     const lowestTicketPriceInSection = ticketGroups.map(({ price }) => price).sort((a, b) => a - b)[0]
-    const percentile = this.sortedTicketGroupPrices.indexOf(lowestTicketPriceInSection) / this.sortedTicketGroupPrices.length
-    const sectionPercentileKeys = Object.keys(sectionPercentiles).map(key => +key).sort()
-    for (const key of sectionPercentileKeys) {
-      if (percentile <= key) {
-        return sectionPercentiles[key]
+
+    const ranges = $costRanges(this.state, this.props)
+
+    for (const range of ranges) {
+      if (range.max > lowestTicketPriceInSection) {
+        return range.color
       }
     }
+
+    return ranges[ranges.length - 1]
```

Comment:
I don't believe this is the same return value as `range.color`. Might be helpful to add types to this function (both method signature and the code within it).